static/css/components/read-panel.less

```diff
@@ -37,7 +37,11 @@
     text-align: center;
   }
 
-  strong,
+  strong {
```

Comment:
this was causing `<strong>` tags to display block (which was breaking waitlist messaging in read-panel)

Reply:
Not sure why linter isnt complaining about this